site/docs/index.md

```diff
@@ -5,62 +5,34 @@ title: Home
 
 
 
+## What is Substrait?
 
+Substrait is a format for describing compute operations on structured data. It is designed for interoperability across different languages and systems.
 
-## Project Vision
 
-Create a well-defined, cross-language [specification](spec/specification) for data compute operations. This includes a declaration of common operations, custom operations and one or more serialized representations of this specification. The spec focuses on the semantics of each operation and a consistent way to describe.
 
-In many ways, the goal of this project is similar to that of the Apache Arrow project. Arrow is focused on a standardized memory representation of columnar data. Substrait is focused on what should be done to data.
+## How does it work?
 
+Substrait provides a well-defined, cross-language [specification](spec/specification) for data compute operations.  This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.  The spec focuses on the semantics of each operation.  In addition to the format and specification, the Substrait community also includes a number of libraries and other [useful tools to get started](/tools/producer_tools/).
```

Comment:
> This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.

I find this sentence a little hard to follow.  Maybe shorter sentences?

```
Substrait allows for common operations to be expressed consistently in different systems.  It also contains many extension points for expressing custom operations.  The specification can be serialized into various formats.
```

> In addition to the format and specification

What is "the format"?  I'm not sure that is defined anywhere.

Reply:
Updated the text a bit, I hope I got what you were looking at.